src/Chainweb/Pact/PactService.hs

```diff
@@ -775,7 +775,7 @@ applyPactCmds
     -> Miner
     -> PactServiceM cas (Vector HashCommandResult)
 applyPactCmds isGenesis env cmds miner =
-    V.fromList . sfst <$> V.foldM f mempty cmds
+    V.fromList . reverse . sfst <$> V.foldM f mempty cmds
```

Comment:
lets just use `V.foldrM` - then we can do away with the `reverse` call

Reply:
There isn't one!